internal/pkg/feature/feature_test.go

```diff
@@ -0,0 +1,151 @@
+package feature_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/influxdata/flux/internal/pkg/feature"
+)
+
+func Test_feature(t *testing.T) {
+	cases := []struct {
+		name     string
+		flag     feature.Flag
+		values   map[string]interface{}
+		expected interface{}
+	}{
+		{
+			name: "bool happy path",
+			flag: newFlag("test", false),
+			values: map[string]interface{}{
+				"test": true,
+			},
+			expected: true,
+		},
+		{
+			name: "int happy path",
+			flag: newFlag("test", 0),
+			values: map[string]interface{}{
+				"test": int32(42),
+			},
+			expected: int32(42),
+		},
+		{
+			name: "float happy path",
+			flag: newFlag("test", 0.0),
+			values: map[string]interface{}{
+				"test": 42.42,
+			},
+			expected: 42.42,
```

Comment:
I'm seeing the various values used here, but conceptually I'm not really seeing how to fit into my existing understanding of feature flags. Practically, what do these values represent?